packages/agreement-process/src/routers/AgreementRouter.ts

```diff
@@ -328,18 +330,22 @@ const agreementRouter = (
       const ctx = fromAppContext(req.ctx);
 
       try {
-        validateAuthorization(ctx, [ADMIN_ROLE]);
+        validateAuthorization(ctx, [ADMIN_ROLE, M2M_ADMIN_ROLE]);
+
+        const { data: agreement, metadata } =
+          await agreementService.createAgreement(
```

Comment:
@Carminepo2 one last thing: in the API Spec, the create agreement call is missing the `operationId`

Reply:
I checked in the `agreementApi.yml` spec and the `operationId` for the `createAgreement` call is already present